### ISSUE TITLE
Does not work in IE

### ISSUE THREAD

**Issue by uvconnects** [michael-lang/sample-ng2-mvc#1]
Angular2 has been such a problem in IE. Is there a way to resolve this?

**Comment by michael-lang** [comment]
I'm not sure what issues you are referring to.  Does the issue surface in this project?    My angular2 / angular4 projects work fine in IE 11 and Edge.

If I found any issues I would report to the specific angular project that caused the issue. (Angular, Angular Material, NGRX store, ...whatever)

**Comment by uvconnects** [comment]
Ok thanks, I found on your main page sorry it wasn't you it was
By Gavin Raine REPLY

Great article and really helpful. Thanks Michael !

At first, I thought I had failed to get this right somehow, but it worked perfectly once I stopped trying to use Internet Explorer. (For reference – I saw the error “Unable to get property ‘apply’ of undefined or null reference”.)

It looks this is an issue that several people have had with Angular CLI & IE 11, so I’m off to look at “polyfills.ts” now.

Just wondering to fix this in IE did anyone have any ideas.
